static/src/js/components/GranuleResultsHighlights/__tests__/GranuleResultsHighlights.test.js

```diff
@@ -77,5 +77,21 @@ describe('GranuleResultsHighlights component', () => {
         expect(enzymeWrapper.find('.granule-results-highlights__item-title').text()).toEqual('producer_granule_id_1')
       })
     })
+
+    describe('when granules have no start time nor end time', () => {
+      test('display not provided instead of date', () => {
+        const { enzymeWrapper } = setup({
+          isLoading: false,
+          isLoaded: true,
+          granules: [{
+            // formattedTemporal is intentionally absent from the granule object
```

Comment:
I appreciate the comment but it might be unnecessary

Reply:
done